man/systemd.link.xml

```diff
@@ -967,6 +967,20 @@
           <xi:include href="version-info.xml" xpointer="v232"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ReceivePacketSteeringCPUMask=</varname></term>
+        <listitem>
+          <para>Configures Receive Packet Steering (RPS) list of CPUs to which RPS may forward traffic.
+          Takes a list of CPU indices or ranges separated by either whitespace or commas. Alternatively,
+          takes a special <literal>all</literal> value in which will include all available CPUs in the mask.
```

Comment:
`takes a special <literal>all</literal> value` -> `takes the special value <literal>all</literal>`